src/prommis/solvent_extraction/sx_model_flowsheet_steady_state.py

```diff
@@ -7,20 +7,28 @@
 )
 from idaes.core.util.model_statistics import degrees_of_freedom as dof
 
+import matplotlib.pyplot as plt
```

Comment:
These imports should be removed - you are not using these anymore.

Reply:
I have removed them